test/elm/reusable/data.cql

```diff
@@ -61,4 +61,30 @@ define fluent function add(a System.Integer, b System.Integer) :
   a + b
 
 define testValue1: 'World'.greet()
-define testValue2: 1.add(2)
\ No newline at end of file
+define testValue2: 1.add(2)
+
+
+// @Test: CommonLib3
+library Common3
+using Simple version '1.0.0'
+context Patient
+
+define function ExpensiveFunction():
+  First(expand Interval[1, 100000])
+
+define ExpensiveStatement:
+  ExpensiveFunction()
+
+define ExpensiveStatementRef:
+  ExpensiveStatement union 
+  ExpensiveStatement union 
+  ExpensiveStatement union 
+  ExpensiveStatement union 
+  ExpensiveStatement
+
+// @Test: Using CommonLib3
+using Simple version '1.0.0'
+include Common3 called common3
+context Patient
+
+define ExpensiveStatementRef: common3.ExpensiveStatementRef
```

Comment:
This isn't used in the testing at all. Can it be removed? If it can, the `Common3`.

Otherwise if you wanted to test to make sure this still works with the reference to the other library, make a test similar to the already created one for `CommonLib3` which runs the test from this library. Setting up the sinon spy should still be possible, just need find the library from the context of the main one.